Dockerfile

```diff
@@ -1,42 +1,51 @@
-FROM debian:jessie
+FROM alpine:3.3
```

Comment:
Depends on `OS-5184`, as discussed elsewhere. We should ticket a separate issue to update this when `OS-5184` is shipped.